src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/Contracts/BackupConfigInfo.cs

```diff
@@ -0,0 +1,24 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+using System.Collections;
+using System.Collections.Generic;
+using Microsoft.SqlTools.ServiceLayer.Admin.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.DisasterRecovery.Contracts
+{
+    /// <summary>
+    /// Provides database info for backup.
+    /// </summary>
+    public class BackupConfigInfo: DatabaseInfo
```

Comment:
>DatabaseInfo [](start = 35, length = 12)

I think BackupConfigInfo should contain an instance of a database info object, instead of derive from it.  Is there a reason that a BackupConfigInfo is a DatabaseInfo object?